sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SyntaxTrackers/CSharpObjectInitializationTracker.cs

```diff
@@ -212,8 +212,11 @@ private bool IsLaterAssignedWithAllowedValue(ObjectCreationExpressionSyntax obje
 
             var variableSymbol = GetAssignedVariableSymbol(objectCreation, semanticModel);
 
+            var nextStatements = GetNextStatements(statement);
```

Comment:
`IsLaterAssignedWithAllowedValue` is used to exclude FPs, so including more statements to check can only lead to less noise - therefore it should be safe